### PR TITLE
Makes Corvids more Corvid.

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Changes language syllables, changes speech sounds, nerfs speed, changes random name generator.

Sound credits to Inchadney and Benboncan of freesound.org

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/station/uplifts.dm [hunk 1]
```diff
@@ -112,7 +112,7 @@
 	damage_mask = 'icons/mob/human_races/masks/dam_mask_corvid.dmi'
 	blood_mask = 'icons/mob/human_races/masks/blood_corvid.dmi'
 
-	slowdown = -1
+	slowdown = -0.5
```
Comment (Zuhayr) [inline code/modules/mob/living/carbon/human/species/station/uplifts.dm]: Partial numbers are meaningless as a slowdown. -1 is the lowest possible buff.

File: code/modules/mob/language/uplifts.dm [hunk 2]
```diff
@@ -24,24 +24,26 @@
...
 	colour = "alien"
 	key = "v"
 	flags = WHITELISTED
 	space_chance = 50
 	syllables = list(
-			"ca", "ra", "ma", "sa", "na", "ta", "la", "sha", "scha", "a", "a",
-			"ce", "re", "me", "se", "ne", "te", "le", "she", "sche", "e", "e",
-			"ci", "ri", "mi", "si", "ni", "ti", "li", "shi", "schi", "i", "i"
+			"ti","hi","ki","ya","ta","ha","ka","ya","chi","cha","kah",
+			"skre","ahk","ek","rawk","kraa","ii","kri","ka"
```
Comment (Zuhayr) [inline code/modules/mob/language/uplifts.dm]: Oh god is this vox pidgin's syllable list?

Comment (ParadoxSpace) [reply code/modules/mob/language/uplifts.dm]: yes, I thought it'd be better than whatever was there before (Which sounded more like a bird whispering in your ear or something to that effect)

File: code/modules/mob/living/carbon/human/species/station/uplifts.dm (hunk 1 above)
Comment (ParadoxSpace) [reply code/modules/mob/living/carbon/human/species/station/uplifts.dm]: fixed

File: code/modules/mob/language/uplifts.dm (hunk 2 above)
Comment (Zuhayr) [reply code/modules/mob/language/uplifts.dm]: Neo corvid vox confirmed.

**Pull request merged**